package.json

```diff
@@ -44,7 +44,7 @@
     "web3modal": "^1.9.1"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "PORT=3001 react-scripts start",
```

Comment:
You can use either a local .env file, or export this variable in your .bashrc | .profile